blog/views.py

```diff
@@ -21,18 +21,22 @@ class PostForm(forms.Form):
         (Privacy.FRIENDS, "Friends only"),
         (Privacy.FOAF, "Friends of friends"),
     )))
+    image = forms.ImageField(label = 'Choose an Image')
     content_type = forms.CharField(widget=forms.Select(choices=(
         ("text/plain", "Plain Text"),
         ("text/markdown", "Markdown"),
     )))
```

Comment:
we should use javascript to hide the text box and unhide the image selection based on content type. our current models has posts being either text or image, not both

Reply:
For now, the post can be either Plain Text + Image or Markdown.

Reply:
wouldn't it be easier to just add a content-type? the api spec seems to require that images have their own content type, and that the content is either plain text or an image

Reply:
okay, I have separated plaintext and image